### ISSUE TITLE
keystone.Email headers

### ISSUE THREAD

**Issue by hatzipanis** [keystonejs/keystone#2687]
Are we able to edit the headers when sending email via `keystone.Email`? I would like to add a `Reply-To` and `Sender` but can't seem to figure out how to pass this to the `keystone.Email` object. 

```
new keystone.Email('booking-notification').send({
	to: managerEmailAddress[booking.restaurant],
	from: {
		name: booking.name,
		email: 'testing@bookings.co.nz' // Verified domain
	},
	subject: subject,
	booking: booking
}, callback);
```

**Comment by jossmac** [comment]
The keystone `Email` prototype needs to be updated to accept the `headers` option. Then it will be like:
```js
keystone.Email('...').send({
  ...
  headers: {
    'Reply-To': 'email@example.com',
  },
});
```

Given the [recent changes to Mandrill](http://blog.mandrill.com/important-changes-to-mandrill.html) I think this deserves a patch.

@mxstbr the options are being [set explicitly](https://github.com/keystonejs/keystone/blob/v0.3.x/lib/email.js#L479) in the email prototype rather than extending them. Is this something you could take a look at?

Also ping @JedWatson

**Comment by mxstbr** [comment]
On it

**Comment by mxstbr** [comment]
Fixed by @jossmac in ae2ba02.
